recipe/meta.yaml

```diff
@@ -12,8 +12,8 @@ package:
 
 # Reminder: source should point to path for repo or github archive for feedstock
 source:
-  url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
-  sha256: fde86223dd405ea0956a28ce9f4f6b201dcaa5921d9be47b72d078cae24f4cfa
+  url: https://github.com/uxlfoundation/scikit-learn-intelex/archive/{{ version }}.tar.gz
+  sha256: 3f49de035da70ea036e6dff932bc3cba3ea2b9343ea858e000bc2265e866ddf7
```

Comment:
Where does this point? Is it meant to point to the latest commit of 2025.5 release branch (ie c3ef708ce0f0023977b6993436d69e39babfa142)